source/Octopus.Client/Editors/Async/MachineEditor.cs

```diff
@@ -54,20 +55,29 @@ public MachineEditor(IMachineRepository repository)
             EnvironmentResource[] environments,
             string[] roles,
             TenantResource[] tenants,
-            TagResource[] tenantTags)
+            TagResource[] tenantTags, 
+            TenantedDeploymentParticipation? tenantedDeploymentParticipation = null)
```

Comment:
if we _dont_ allow it to be nullable, then is that a problem? Wont old servers just ignore the property.